src/ImageSharp/Common/Helpers/RuntimeEnvironment.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace SixLabors.ImageSharp
+{
+    /// <summary>
+    /// Provides information about the .NET runtime installation.
+    /// Many methods defer to <see cref="RuntimeInformation"/> when available.
+    /// </summary>
+    internal static class RuntimeEnvironment
+    {
+        private static readonly Lazy<bool> IsNetCoreLazy = new Lazy<bool>(() => FrameworkDescription.StartsWith(".NET Core", StringComparison.OrdinalIgnoreCase));
+
+        /// <summary>
+        /// Gets a value indicating whether the .NET installation is .NET Core
+        /// </summary>
+        public static bool IsNetCore => IsNetCoreLazy.Value;
```

Comment:
I guess this definition of ".NET Core" excludes .NET 5. We need to be more careful here, the terminology already became ambiguous. (Same product rebranded TBH ...)

```suggestion
        /// <summary>
        /// Gets a value indicating whether the .NET installation is .NET Core 3.1 or lower.
        /// </summary>
        public static bool IsNetCoreClassic => IsNetCoreLazy.Value;
```

Reply:
I would probably call other querying methods. 

- `IsNet`
- `IsNetFramework`

and flesh out the intellisense docs. .NET 5 returns `.NET` as the prefix, while < .NET 4.8 return `.NET Framework`. 

It's a crappy solution anyway. .NET Core was and always shall be a terrible name.

Reply:
At least let's change the comment for now to avoid shooting ourselves in the feet!

```suggestion
        /// <summary>
        /// Gets a value indicating whether the .NET installation is .NET Core 3.1 or lower.
        /// </summary>
        public static bool IsNetCore => IsNetCoreLazy.Value;
```